src/secret_connection.rs

```diff
@@ -1,4 +1,4 @@
-use byteorder::{BigEndian, ByteOrder};
+use byteorder::{ByteOrder, LittleEndian};
```

Comment:
FYI, if you find `LittleEndian` and `BigEndian` too verbose there are also `BE` and `LE` aliases.